src/main/main.ts

```diff
@@ -291,10 +300,19 @@ ipcMain.on('settingsWindow', (event, args) => {
   }
     
   if (settingsWindow === null) return; 
-
+  
   if (args[0] === "quit") {
     console.log("User closed settings");
     settingsWindow.close();
+    storageDir = getPathConfigSafe(cfg, 'storage-path');
+    baseLogPath = getPathConfigSafe(cfg, 'log-path');
+    maxStorage = getNumberConfigSafe(cfg, 'max-storage');
+    if (checkConfig()) {
+      updateStatus(0);
+      recorder = new Recorder(storageDir, maxStorage);  
```

Comment:
Can you merge main into this branch? I think when you do that this should have a 3rd 'monitor index' argument of the integer 0. Is stuff I added tonight.

Reply:
really odd its not telling me thats a merge conflict

Reply:
yeah fuck knows why not - I tried to merge in main myself but it didn't work so I immediately gave up. We'll just deal with it if there is a problem.